src/lib/browser/Page.ts

```diff
@@ -200,6 +201,11 @@ export class BrowserPage {
           await req.abort();
           return;
         }
+        if (adblock && adblocker.match(new URL(reqUrl))) {
```

Comment:
A function `isRequestBlocked` might be nicer

Reply:
to wrap `adblock && adblocker.match(new URL(reqUrl))` or renaming `match` to `isRequestBlocked`?

Reply:
I mean to wrap all the blocking tests, like the one juste above also, to avoid repeating `await req.abort()`

Reply:
ah true, we can abstract now that the function has grown quite a lot 👌🏻 
I'll do that in a futur PR